hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueService.java

```diff
@@ -365,14 +367,8 @@ public void rollbackTransaction(String transactionId) {
     }
 
     @Override
-    public Map<String, LocalQueueStats> getStats() {
-        Map<String, LocalQueueStats> queueStats = createHashMap(containerMap.size());
-        for (Entry<String, QueueContainer> entry : containerMap.entrySet()) {
-            String name = entry.getKey();
-            LocalQueueStats queueStat = createLocalQueueStats(name);
-            queueStats.put(name, queueStat);
-        }
-        return queueStats;
+    public void probeIn(ProbingCycle cycle) {
+        Probing.probeIn(cycle, "queue", statsMap);
```

Comment:
This interaction with a static utility class makes the api feel not very intuitive. Best to move this method to the ProbingCycle and perhaps work with an abstract class implementation for now instead of an interface; so we can easily replace it by default methods and back to an interface when switching to Java 8.